File: 240510/특정 숫자 맞추기/catching-specific-number.js
const fs = require('fs');
const nums = fs.readFileSync(0).toString().trim().split('\n').map(Number);

let idx = 0;
while(true){
    if(nums[idx] === 25) break;

    if(nums[idx] < 25) console.log("Higher");
    else console.log("Lower");

    idx++;
}

console.log("Good");

File: 240510/특정 숫자 맞추기/catching-specific-number.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int score;
        while((score = sc.nextInt()) != 25){
            if(score < 25){
                System.out.println("Higher");
            }else {
                System.out.println("Lower");
            }
        }

        System.out.println("Good");

    }
}